apm/config.go

```diff
@@ -0,0 +1,85 @@
+package apm
+
+type MonitorConfig struct {
+	PopulateEvents bool
+	Commands       []string
+	Databases      []string
+	Collections    []string
+	Namespaces     []Namespace
+}
+
+type Namespace struct {
+	DB         string
+	Collection string
+}
+
+func stringSliceContains(slice []string, item string) bool {
+	if len(slice) == 0 {
+		return false
+	}
+
+	for idx := range slice {
+		if slice[idx] == item {
+			return true
+		}
+	}
+
+	return false
+}
+
+func (c *MonitorConfig) shouldTrack(e eventKey) bool {
+	if c == nil {
+		return true
```

Comment:
Why does `shouldTrack` return `true` if `*MonitorConfig` is nil?

Reply:
because if there isn't a monitoring config defined, then we should track everything. the other option seems unintuitive.